src/commands/snapshot.ts

```diff
@@ -18,18 +18,18 @@ export default class Snapshot extends PercyCommand {
   static examples = [
     '$ percy snapshot _site/',
     '$ percy snapshot _site/ --base-url "/blog"',
-    '$ percy snapshot _site/ --ignore-files "\.(blog|docs)$"',
+    '$ percy snapshot _site/ --ignore-files "/blog/**,**/blog.*"',
```

Comment:
```suggestion
    '$ percy snapshot _site/ --ignore-files "/blog/drafts/**"',
```

Was trying to think up a more legit case for using ignore files.